bindings/go/README.md

```diff
@@ -17,12 +17,38 @@ provided there is enough buffer size to accomodate several of such blocks in one
 Read()/Write() request. (For example, see the source code of `mcp` utility and its `-bsz`
 option.)
 
-To use `go/mio` in your app 1) install motr-devel pkg and 2):
+## Installation
+
+Use go get, make sure you using Go version with [modules](https://github.com/golang/go/wiki/Modules) support.
+
+```Shell
+go get github.com/Seagate/cortx-motr/bindings/go
+```
+
+Next install `motr-devel` pkg and then import into sources.
 
 ```Go
 import github.com/Seagate/cortx-motr/bindings/go/mio
 ```
 
+Then init mio:
+
+```Go
+ha_addr := "192.168.63.52@tcp:12345:34:1"
+local_addr := "192.168.63.52@tcp:12345:33:1000"
+profile := "0x7000000000000001:0"
+process_fid := "0x7200000000000001:64"
+mio.Init(local_addr, ha_addr, profile, process_fid)
+```
+
+To build you project, you need to set `CGO_CFLAGS` and `CGO_LDFLAGS` cgo environment as below, this will override `CFLAGS` and `LDFLAGS` in `mio.go`.
+
+```Shell
+export MOTR_HOME=/home/cortx-motr
+CGO_CFLAGS="-I/$MOTR_HOME -I/usr/include/motr -DM0_EXTERN=extern -DM0_INTERNAL= -Wno-attributes" \
+CGO_LDFLAGS="-L$MOTR_HOME/motr/.libs -Wl,-rpath=$MOTR_HOME/motr/.libs -lmotr" go build
```

Comment:
This all should not be needed in motr is installed from rpms, please check.

Reply:
This is for project which import `github.com/Seagate/cortx-motr/bindings/go/mio`, because they might have difference motr home location then `mio.go` which currently CFLAGS point to relative directory of motr because of the file location of `mio.go`

Reply:
It's specified above that motr and motr-devel packages should be installed. If the user does so, setting these variables is not needed, right?